lib/tasks_private/openstack.rake

```diff
@@ -34,4 +34,65 @@ namespace :vcr do
     Rake::Task['vcr:spec:run'].invoke
     Rake::Task['vcr:credentials:obfuscate'].invoke
   end
+
+  task :rerecord_v2 => [] do
```

Comment:
Looking for suggestions to have a better name for the rake task

Reply:
We can just replace the old `vcr:rerecord` task competely.  Using Rails secrets and VCR `config.define_cassette_placeholder` we don't need the credentials:load and credentials:obfuscate at all

Reply:
@agrare Does that mean we want to remove the old dependent tasks as well? Such as `lib/tasks_private/spec.rake` and `lib/tasks_private/vcr_recorder.rb`